cms/templates/js/course-video-settings.underscore

```diff
@@ -12,7 +12,7 @@
         <div class='course-video-settings-message-wrapper'></div>
         <span class="course-video-settings-title"><%- gettext('Transcript Settings') %></span>
         <div class='transcript-preferance-wrapper transcript-provider-wrapper'>
-            <label class='transcript-preferance-label' for='transcript-provider'><%- gettext('Transcript Provider') %><span class='error-icon' aria-hidden="true"></span></label>
+            <label class='transcript-preferance-label' for='transcript-provider'><%- gettext('Course Video Settings') %><span class='error-icon' aria-hidden="true"></span></label>
```

Comment:
You've changed the wrong text. We need to change the text at line 13 of this.

Reply:
okay